src/js/containers/explorer/detail/table/ExplorerTableContainer.jsx

```diff
@@ -0,0 +1,259 @@
+/**
+ * ExplorerTableContainer.jsx
+ * Created by Lizzie Salita 10/16/17
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { orderBy, range } from 'lodash';
+
+import * as MoneyFormatter from 'helpers/moneyFormatter';
+import * as explorerActions from 'redux/actions/explorer/explorerActions';
+
+import ExplorerTableFields from 'dataMapping/explorer/explorerTableFields';
+
+import ExplorerTable from 'components/explorer/detail/visualization/table/ExplorerTable';
+
+const propTypes = {
+    order: PropTypes.object,
+    pageNumber: PropTypes.number,
+    setExplorerTablePage: PropTypes.func,
+    results: PropTypes.object,
+    goDeeper: PropTypes.func,
+    total: PropTypes.number
+};
+
+export class ExplorerTableContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            columns: [],
+            results: [],
+            pageOfItems: [],
+            pager: {}
+        };
+
+        this.onChangePage = this.onChangePage.bind(this);
+    }
+
+    componentDidMount() {
+        this.showColumns();
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.results !== this.props.results) {
+            this.parseResults(nextProps.results);
+        }
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.order !== prevProps.order) {
+            // table sort changed
+            this.orderResults();
```

Comment:
When does `orderResults` get called when the initial data set is loaded? It only seems to be sorted once the order prop changes.